apps/studio/pages/project/[ref]/database/tables/[id].tsx

```diff
@@ -50,11 +51,17 @@ const DatabaseTables: NextPageWithLayout = () => {
         </ScaffoldSection>
       </ScaffoldContainer>
 
-      <DeleteConfirmationDialogs selectedTable={selectedTable} />
-      <SidePanelEditor
-        includeColumns
-        selectedTable={isTableLike(selectedTable) ? selectedTable : undefined}
-      />
+      {project?.ref !== undefined && selectedTable !== undefined && isTableLike(selectedTable) && (
+        <TableEditorTableStateContextProvider
+          key={`table-editor-table-${selectedTable.id}`}
+          projectRef={project?.ref}
+          table={selectedTable}
+          editable={true}
```

Comment:
likewise here - needs proper permission checking?

Reply:
@alaister want me to do this?
think these were your changes, not sure what you were planning